src/NHibernate.Config.Templates/Oracle-Managed.cfg.xml

```diff
@@ -14,5 +14,6 @@ for your own use before compile tests in VisualStudio.
 		<property name="show_sql">false</property>
 		<property name="dialect">NHibernate.Dialect.Oracle10gDialect</property>
 		<property name="query.substitutions">true 1, false 0, yes 'Y', no 'N'</property>
+		<property name="oracle.use_n_prefixed_types_for_unicode">false</property>
```

Comment:
For helping those installing Oracle with an non-Unicode default charset, I have added this parameter in the sample file. So they will just need to switch it to true, as explained in the instruction file in the same folder.